### PR TITLE
centralised logging via lager and journal

### PR DESCRIPTION
**Pull request by ContentSafe**

### CODE REVIEW

**Comment by ContentSafe** [comment]
review of the readmes will be done by holger

File: src/eradius_log.erl [hunk 1]
```diff
@@ -122,12 +134,25 @@ format_cmd(discreq)   -> <<"Disconnect-Request">>;
 format_cmd(discack)   -> <<"Disconnect-Ack">>;
 format_cmd(discnak)   -> <<"Disconnect-Nak">>.
 
+format_cmd_string(request)   -> "Access-Request";
```
Comment (ContentSafe) [inline src/eradius_log.erl]: duplication is going to be removed in later releases, for now i wanted to avoid unnecessary conversions from binaries

**Comment by RoadRunnr** [comment]
could please cleanup the commit mess, it is not necessary to keep every single interim step in the history. Each commit should be address one specific concern.

**Comment by ContentSafe** [comment]
working on it :)

**Comment by ContentSafe** [comment]
here we go, i think 2 merge conflicts will happen, but they should be easy to solve

**Comment by RoadRunnr** [comment]
please rebase your tree to avoid the merge conflics

File: src/eradius_log.erl (hunk 1 above)
Comment (RoadRunnr) [inline src/eradius_log.erl]: what? lager should be taking binaries as strings just fine

**Comment by ContentSafe** [comment]
after rebase the only interesting commits are the last 3 one (whereas the last one just fixes some issues that happened via reformating and removing of unused code)

**Comment by ContentSafe** [comment]
the two mayor points - switch from eradius to lager and informativ lager logging - are now in two separate commits.
four minor changes also separated thematically into commits.

**Comment by ContentSafe** [comment]
closed, this issues were covered in https://github.com/travelping/eradius/pull/5 and https://github.com/travelping/eradius/pull/6

**Pull request closed without merging**